### DISCUSSION TITLE
GPS based Photos/timelapse

### DISCUSSION THREAD

**Discussion by DustinS123** [gopro/labs#1005] [Q&A]
Hello everyone, I need a QR code or a script that automatically triggers the Gopro Hero 10 at a defined distance every 5 meters base on the GPS to takes a picture. I'm still very new to Gopro Labs and would be grateful for any explanation.

Best regards

Dustin

**Comment by 0dan0** [comment]
I've done that, but it has been a long while. The code would need to be tested on HERO10.

Try:  ```mP!D>c:A5$FRMA=0,0+!S!R```

```mP``` - mode Photo
```!D``` - Wait until DOP is 20 or better (GPS precision)
```>c:A5```  - if(distance_point_A > 5 meters)
```   $FRMA=0,0```  -  reset pointA
```   +!S``` - and Start photo capture
```!R``` - repeat everything

**Reply by DustinS123** [reply]
Thank you !

**Reply by DustinS123** [reply]
ist it the same skript for the Hero 8 Black ?

**Reply by 0dan0** [reply]
No, HERO8 is too old for this type of scripting.
